package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lnd-engine",
-  "version": "0.5.4-beta-rc2",
+  "version": "0.5.5-beta-rc1",
```

Comment:
Do you want to do the bump here or in a separate PR? I thought in general we were doing these in separate PRs.

Reply:
@treygriffith we should bump the version now because you'll have conflicts when working w/ older images of the broker vs. newer images of the relayer. It makes it easier for the time being, while the broker is on an older lnd version

Reply:
sorry, i misread. I can bump the version in a different PR if you wanted